packages/braid-design-system/src/lib/components/Tiles/Tiles.tsx

```diff
@@ -20,106 +15,46 @@ export interface TilesProps {
-   * */
-  dividers?: boolean | DividerProps['weight'];
   data?: DataAttributeMap;
 }
 
 export const Tiles = ({
   children,
   space = 'none',
   columns = 1,
-  dividers,
   data,
   ...restProps
-}: TilesProps) => {
-  if (process.env.NODE_ENV !== 'production') {
-    if (typeof dividers !== 'undefined') {
-      // eslint-disable-next-line no-console
-      console.warn(
-        dedent`
-          The "dividers" prop is deprecated and will be removed in v33 to enable CSS gap in layout components.
-          Update your usage now to make upgrading easier by manually inserting "Divider" components as required:
-            %c- <Tiles space="..." dividers>
-            %c+ <Tiles space="...">
-            %c    <Component>{item}</Component>
-            %c+   <Stack space="...">
-            +     <Hidden above="mobile"><Divider /></Hidden>
-            %c      <Component>{item}</Component>
-            %c+   </Stack>
-            +   <Stack space="...">
-            +     <Hidden above="mobile"><Divider /></Hidden>
-            %c      <Component>{item}</Component>
-            %c+   </Stack>
-            %c  </Tiles>
-          See migration guide for details: https://github.com/seek-oss/braid-design-system/blob/master/docs/Removing%20dividers%20support%20from%20layout%20components.md
-        `,
-        'color: red',
-        'color: green',
-        'color: inherit',
-        'color: green',
-        'color: inherit',
-        'color: green',
-        'color: inherit',
-        'color: green',
-        'color: inherit',
-      );
-    }
-  }
-
-  return (
+}: TilesProps) => (
+  <Box
+    className={negativeMargin('top', space)}
+    {...buildDataAttributes({ data, validateRestProps: restProps })}
+  >
     <Box
-      className={negativeMargin('top', space)}
-      {...buildDataAttributes({ data, validateRestProps: restProps })}
+      display="flex"
+      flexWrap="wrap"
+      className={negativeMargin('left', space)}
     >
-      <Box
-        display="flex"
-        flexWrap="wrap"
-        className={negativeMargin('left', space)}
-      >
-        {Children.map(flattenChildren(children), (child, i) => (
+      {Children.map(flattenChildren(children), (child) => (
+        <Box
+          minWidth={0}
+          className={resolveResponsiveProp(
+            columns,
+            styles.columnsMobile,
+            styles.columnsTablet,
+            styles.columnsDesktop,
+            styles.columnsWide,
+          )}
+        >
           <Box
-            minWidth={0}
-            className={resolveResponsiveProp(
-              columns,
-              styles.columnsMobile,
-              styles.columnsTablet,
-              styles.columnsDesktop,
-              styles.columnsWide,
-            )}
+            height="full"
+            // This needs to be a separate element to support IE11.
```

Comment:
Can we collapse this element now that we don't support IE11?

Reply:
Deliberately left this as divider removal only, as a subsequent PR refactors the `Tiles` component to use CSS grid.